tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -90,7 +90,7 @@ std::vector<std::shared_ptr<T>> extractMeshData(const aiScene* scene,
   {
     auto vertices = std::make_shared<tesseract_common::VectorVector3d>();
     auto triangles = std::make_shared<Eigen::VectorXi>();
-    std::shared_ptr<tesseract_common::VectorVector3d> normals = nullptr;
+    std::shared_ptr<tesseract_common::VectorVector3d> normals_ = nullptr;
```

Comment:
Maybe rename to `vertex_normals` to align with ROS Style guide?

Reply:
I've made the change locally but am waiting for the CI to finish before pushing.